subsys/usb/usb_c/usbc_pe_common.c

```diff
@@ -858,21 +854,22 @@ static void pe_drs_send_swap_run(void *obj)
 			 * a Type-C Error Recovery.
 			 */
 			usbc_request(dev, REQUEST_TC_ERROR_RECOVERY);
-			return;
+			return SMF_EVENT_PROPAGATE;
 		}
 
 		/* return to ready state */
 		pe_set_ready_state(dev);
-		return;
+		return SMF_EVENT_PROPAGATE;
 	} else if (atomic_test_and_clear_bit(pe->flags, PE_FLAGS_MSG_DISCARDED)) {
 		/*
 		 * Inform Device Policy Manager that the message
 		 * was discarded
 		 */
 		policy_notify(dev, MSG_DISCARDED);
 		pe_set_ready_state(dev);
-		return;
+		return SMF_EVENT_PROPAGATE;
```

Comment:
both of these returns seem unneeded, as the same return is directly after the ifs end